app/views/layouts/_google_analytics_script.haml

```diff
@@ -1,6 +1,14 @@
 - if (APP_CONFIG.use_google_analytics)
   <script type="text/javascript">
   - if APP_CONFIG.use_google_tag_manager.to_s == "true"
+    var dataLayer = dataLayer || [];
+    (function(dl){
+    = "var person_emails = [#{person_emails(@current_user).map { |e| "'"+e+"'" }.join(',')}];"
```

Comment:
This is horrible, how could I do it otherwise? I want an array of strings, e.g. `["mikko.porkola@sharetribe.com"]`. With `person_emails(@current_user).to_json` I get
`[&quot;mikko.porkola@sharetribe.com&quot;]` which isnt' exactly doesn't work.